tests/unit/test_bo.py

```diff
@@ -111,12 +111,21 @@ def test_acquisition():
     acq_noise_cov = np.random.rand(n_params, n_params) * 0.5
     acq_noise_cov += acq_noise_cov.T
     acq_noise_cov += n_params * np.eye(n_params)
-    t = 1
+    with pytest.raises(ValueError):
+        acquisition.LCBSC(target_model, noise_var=acq_noise_cov)
+
+    # check acquisition with negative variances
+    acq_noise_var = -0.1
+    with pytest.raises(ValueError):
+        acquisition.LCBSC(target_model, noise_var=acq_noise_cov)
+
+    acq_noise_var = {'a': 0.1, 'b': -0.1}
     with pytest.raises(ValueError):
         acquisition.LCBSC(target_model, noise_var=acq_noise_cov)
```

Comment:
same as above, we set `acq_noise_var` but test `noise_var=acq_noise_cov`

Reply:
ah, now I see what you mean.